.github/workflows/cicd.yml

```diff
@@ -27,9 +27,13 @@ jobs:
     strategy:
       matrix:
         # supported scala versions
-        scala:
-          - 2.12.8
-          - 2.13.4
+        include:
```

Comment:
We have to fix the build name to not impact on mergify rules at every changes. 

This should be something like `build(VERSION)`, currently also task are included in the name